release-notes/topics/aap-212.adoc

```diff
@@ -0,0 +1,56 @@
+[[aap-2.1.2-intro]]
+= Ansible Automation Platform 2.1.2
+ 
+Red Hat Ansible Automation Platform simplifies the development and operation of automation workloads for managing enterprise application infrastructure lifecycles. It works across multiple IT domains including operations, networking, security, and development, as well as across diverse hybrid environments. Simple to adopt, use, and understand, Red Hat Ansible Automation Platform provides the tools needed to rapidly implement enterprise-wide automation, no matter where you are in your automation journey.
+ 
+.Enhancements
+* Added architecture support labels for the Ansible Automation Platform operator
+* Updated to Receptor 1.2.1 on all supported versions of the Ansible Automation Platform
+* Removed unnecessary packages from the bundle installer to reduce the file size
+* Updated to version to Ansible Runner 2.1.3
+* Removed the enforcement of license compliance from the Ansible Automation Platform installer
+* Added support to enable an array of pull secrets to pass to the operator
+* Added a preflight check to ensure that the installer is running on a UTF-8 system
+* Removed unnecessary `become_user:root` entries from the Ansible Automation Platform installer
+* Added support to provide a custom secret key when running the `rekey.yml` file
+* Added a preflight check to detect missing username and password variables for service catalog worker, instead of causing an installation failure at a later point of time
+* Updated the collection counts on the *Content* tab when the collection details were filtered
+ 
+.Bug fixes
+* Fixed an issue where Controller user interface would not load on a few cloud providers
+* Added compatibility with earlier versions for `image_pull_secret` key in `awx-operator`, so that the backup and restore operations are completed successfully
+* Automation controller backups no longer fail when multiple secrets are supported
+* Resolved certification mismatch during upgrade so that every controller dashboard is accessible
+* Improved the labels for Japanese translations
+* Resolved issues when upgrading automation hub from version 2.1.1 to version 2.1.2
+* Resolved issues where the Controller user interface was not accessible, when automation controller was installed through Operator on OpenShift Core Platform 4.10
+* Fixed an issue that caused the upgrade from Ansible Automation Platform 1.2 to 2.1 to fail, when the execution environment was transferred to automation hub
+* Fixed an issue so that a standalone installation of private automation hub can now transfer the execution environments to the hub
```

Comment:
Does this correspond to https://issues.redhat.com/browse/AAH-1373 ticket?

Reply:
I believe this corresponds to https://issues.redhat.com/browse/AAP-2112. I have added another note to reference https://issues.redhat.com/browse/AAH-1373 below.